packages/sysutils/amremote/scripts/remote-config

```diff
@@ -18,28 +18,27 @@ fi
 if [ -n "$SOURCEREMOTECONF" ]; then
   cp $SOURCEREMOTECONF/remote*.conf /tmp
   if [ "$REMAP" = "yes" ]; then
-    sed -i 's/ 15[^0-9]*$/ 1/
-            s/ 63[^0-9]*$/ 90/
-            s/ 97[^0-9]*$/ 28/
-            s/ 102[^0-9]*$/ 172/
-            s/ 125[^0-9]*$/ 46/
-            s/ 128[^0-9]*$/ 45/
-            s/ 139[^0-9]*$/ 46/
-            s/ 142[^0-9]*$/ 116/
-            s/ 143[^0-9]*$/ 116/
-            s/ 158[^0-9]*$/ 1/
-            s/ 183[^0-9]*$/ 59/
-            s/ 184[^0-9]*$/ 399/
-            s/ 185[^0-9]*$/ 400/
-            s/ 186[^0-9]*$/ 60/
-            s/ 232[^0-9]*$/ 28/
-            s/ 240[^0-9]*$/ 164/
-            s/ 241[^0-9]*$/ 163/
-            s/ 242[^0-9]*$/ 165/
-            s/ 244[^0-9]*$/ 208/
-            s/ 245[^0-9]*$/ 168/
-            s/ 264[^0-9]*$/ 63/
-            s/ 704[^0-9]*$/ 116/' /tmp/remote*.conf
+    sed -i 's/ 15[^0-9]*$/ 1		# KEY_TAB -> KEY_ESC/
+            s/ 63[^0-9]*$/ 90		# KEY_F5 -> KEY_KATAKANA/
+            s/ 97[^0-9]*$/ 28		# KEY_RIGHTCTRL -> KEY_ENTER/
+            s/ 102[^0-9]*$/ 172	# KEY_HOME -> KEY_HOMEPAGE/
+            s/ 125[^0-9]*$/ 127	# KEY_LEFTMETA -> KEY_COMPOSE/
```

Comment:
This part removes <c>Playlist</c> functionality for FullscreenVideo and
<c>ActivateWindow(PVROSDChannels)</c> funtionality for FullscreenLiveTV

Reply:
Personally I never used Playlist functionality for FullscreenVideo, dunno if it usable.

For `PVROSDChannels`, we still have this one:
```
      <enter mod="longpress">ActivateWindow(PVROSDChannels)</enter>
```
https://github.com/xbmc/xbmc/blob/master/system/keymaps/keyboard.xml#L689